tests/scripts/thread-cert/thread_cert.py

```diff
@@ -465,6 +473,27 @@ def _collect_test_info_after_setup(self):
                 ethaddr = node.get_ether_mac()
                 test_info['ethaddrs'][i] = EthAddr(ethaddr).format_octets()
 
+    def _construct_backbone_network_name(self, backbone_network_id) -> str:
+        """
+        Construct the name of the backbone network based on the given backbone network id from TOPOLOGY. The format
+        is `backbone{PORT_OFFSET}.{backbone_network_id}`.
+
+        Args:
+            backbone_network_id: The backbone network id or None.
+
+        Returns:
+            backbone_name (str): The name of the backbone network.
+
+        Raises:
+            AssertionError: If the constructed backbone network name is not in the list of self._backbone_network_names.
```

Comment:
The code is clear enough so these docs may not be necessary.

Reply:
Python function docstrings are useful for documentation generation (maybe in the future) and IDE integration (IDE help to provide helpful tips and suggestions when coding), so I would prefer to keep it here.

Reply:
will update